tests/functional/configure/test_sso.py

```diff
@@ -69,7 +69,7 @@ def setUp(self):
         self.profile_prompt.start()
 
     def tearDown(self):
-        super(TestConfigureSSOCommand, self).tearDown()
+        super().setUp()
```

Comment:
This should be `tearDown()` instead of `setUp()`.

Reply:
:facepalm: https://github.com/aws/aws-cli/pull/9090/commits/768afeffa8abf7e70a43ef23ca69b17d7b03ab73